section4/reflection.md

```diff
@@ -2,21 +2,82 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
+ Great so far! It has only been a couple of (long) days, so I am interested to see how it pans out over a week or two, but I like the flow it creates. I'm more of a 45 minute session person than 25 minute. I need a little more time to really dig in. And I think a slightly longer break is also better for me. Longer sessions, longer breaks. More like a 5-7 minute break is what I need to refocus again afterward.
```

Comment:
I'm glad you know this about yourself! Make sure when you are prepping for timed events, you are getting in the habit of
1. Focusing on your process (find ways to document - I would typically have a file with my psuedocode and comments and a clean file to "turn in"). Documenting and focusing on the process are super valuable skills as you transition to the job hunt and prep for the coding/technical challenges
2. Practicing coding/working under timed pressure
3. Take your breaks and find ways to shut off your brain for those few moments